source/guides/stacks/mers.rst

```diff
@@ -176,7 +217,7 @@ can be used for reference.
 
    .. code-block:: bash
 
-      docker build --no-cache -t clearlinux/stacks-mers .
+      docker build --no-cache -t sysstacks/mers-clearlinux .
 
 Using the |MERS| container image
```

Comment:
General rule, don't use -ing words in section titles. Instead "Use the MeRS container image"